java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1011,4 +1012,32 @@ public void zcard(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.zcard("foo").get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void type(BaseClient client) {
+        String nonExistingKey = UUID.randomUUID().toString();
+        String stringKey = UUID.randomUUID().toString();
+        String listKey = UUID.randomUUID().toString();
+        String hashKey = UUID.randomUUID().toString();
+        String setKey = UUID.randomUUID().toString();
+        String zsetKey = UUID.randomUUID().toString();
+        // TODO: check with stream once XSET is implemented
+        String streamKey = UUID.randomUUID().toString();
+
+        client.set(stringKey, "value").get();
+        client.lpush(listKey, new String[] {"value"}).get();
+        client.hset(hashKey, Map.of("1", "2")).get();
+        client.sadd(setKey, new String[] {"value"}).get();
+        client.zadd(zsetKey, Map.of("1", 2.)).get();
+
+        assertAll(
```

Comment:
if you can, I think it would be good to check with toLowerCase() 
if you think this creates complex than its ok

Reply:
not hard.  I added an equalsIgnoreCase instead 👍